units/patch_intent_test.go

```diff
@@ -133,14 +145,18 @@ func (s *PatchIntentUnitsSuite) makeJobAndPatch(intent patch.Intent) *patchInten
 }
 
 func (s *PatchIntentUnitsSuite) TestProcessCliPatchIntent() {
+	githubOauthToken, err := s.env.Settings().GetGithubOauthToken()
+	s.Require().NoError(err)
+
 	flags := admin.ServiceFlags{
 		GithubPRTestingDisabled: true,
 	}
 	s.NoError(admin.SetServiceFlags(flags))
 
-	patchContent, err := fetchDiffByURL(s.diffURL)
+	patchContent, err := fetchDiffByURL(&s.githubPatchData, githubOauthToken)
 	s.NoError(err)
 	s.NotEmpty(patchContent)
+	s.NotEqual("{", patchContent[0])
```

Comment:
I assume this is intentional, but just adding a comment here to make sure.

Reply:
yes, that api endpoint can return JSON data if the wrong headers are sent. patchs/diffs either start with "-" or "From" from what i've seen, but I know they don't start with "{".